06-hot-reloading-with-directives/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "example01-combining-local-and-remote-schemas",
+  "version": "0.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start-gateway": "nodemon index.js",
+    "start-service-accounts": "nodemon services/accounts/index.js",
+    "start-service-inventory": "nodemon services/inventory/index.js",
+    "start-service-products": "nodemon services/products/index.js",
+    "start-service-reviews": "nodemon services/reviews/index.js",
+    "start-services": "concurrently \"yarn:start-service-*\""
+  },
+  "dependencies": {
+    "@graphql-tools/schema": "^7.0.0",
+    "@graphql-tools/stitch": "^7.0.4",
+    "@graphql-tools/stitching-directives": "^1.0.0",
```

Comment:
is this expected that new packages don't assume the same base version as the mono repo? I'd expect this to start a v7 to match the Tools version.

Reply:
I actually meant to release as 0.0.1 but encountered a few problems with our automated process because this was a new package and ended up doing a major instead of patch release that gave us 1.0.0 should be 7.0.0 I guess when finished

Reply:
Meaning, I made a mistake as I was trying to debug the automated process...